app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivity.kt

```diff
@@ -129,28 +132,38 @@ fun TravelActivitiesScreen(
               Icon(imageVector = Icons.Default.Add, contentDescription = null)
             }
       }) { pd ->
-        LazyColumn(
-            verticalArrangement = Arrangement.spacedBy(8.dp),
-            contentPadding = PaddingValues(vertical = 8.dp),
-            modifier =
-                Modifier.fillMaxWidth()
-                    .padding(horizontal = 16.dp)
-                    .padding(pd)
-                    .testTag("activityColumn")) {
-              if (listOfActivities.value.isEmpty()) {
-                item {
-                  Text(
-                      text = "No activities planned for this trip",
-                      modifier = Modifier.testTag("emptyTravel"))
-                }
-              } else {
-                items(listOfActivities.value.size) { idx ->
-                  ActivityItem(
-                      listOfActivities.value[idx],
-                      onClick = {
-                        Toast.makeText(context, "Activity clicked", Toast.LENGTH_SHORT).show()
-                      })
-                }
+        Box(
```

Comment:
Couldn't you put the if inside the LazyColumn before the first if-else statement ?

Reply:
No, I'd have to make it a regular item of the lazycolumn. This defeats the intention of having a transparent banner on top of it. Maybe that wasn't what was envisioned perhaps?